features/buyer/catalogue.feature

```diff
@@ -73,3 +73,12 @@ Scenario: User is able to search by keywords field on the search results page to
   Then I see that service.id in the search summary text
   And I see that service.id as the value of the 'q' field
   And I see that service in the search results
+
+Scenario: User is able to click a filter on the search results page to narrow down the results returned
+  Given I am on the /g-cloud page
+  And I have a random g-cloud service from the API
```

Comment:
Will this only give us services that are indexed for search?

Because G-Cloud 8 isn't going to be expired, so if it just returns any live g-cloud service then there's a chance that it will be a G8 and the G8 lots will no longer be filterable.